stack.yaml

```diff
@@ -81,6 +81,12 @@ packages:
     - engine-io
     - engine-io-wai
 
+# Forked version of "dns" with proper Windows support
+- location:
+    git: https://github.com/input-output-hk/dns.git
+    commit: 6da3cd051bd6f6415c8ec3b53f1c0372a80a4376
```

Comment:
Ok, but lets also try and get a minimal set of DNS package changes, suitable to go upstream.

This is OK for now, but next step should be to prepare a clean upstream PR.

Reply:
Aye, I'm on it as we speak (to be precise, I'm waiting for the tests to fully pass to be even surer my fix is hitting the nail on the head).